packages/console/src/Console.tsx

```diff
@@ -974,6 +974,7 @@ export class Console extends PureComponent<ConsoleProps, ConsoleState> {
       >
         <div className="console-pane" ref={this.consolePane}>
           <ConsoleStatusBar
+            dh={dh}
```

Comment:
Need to pull this from context or pass in as prop

Reply:
You can consume a context from a class component btw. It's just complicated if you need to consume multiple contexts. `MyComponent.contextType = Context`

https://legacy.reactjs.org/docs/context.html#classcontexttype